### ISSUE TITLE
QuillSimpleToolbar and  useMaterial3: broken layout on web

### ISSUE THREAD

**Issue by rodrigo-itao** [singerdmx/flutter-quill#1604]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

9.0.1

### Other Flutter Quill packages versions

flutter_quill_extensions: ^9.0.1

### Steps to reproduce

Use QuillSimpleToolbar with ThemeData.light(useMaterial3: false).
Compile to web.

### Expected results

A toolbar with working layout.

### Actual results

A toolbar with broken layout.

### Code sample

<details><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';
import 'package:flutter_quill/flutter_quill.dart';

void main() {
  runApp(MainApp());
}

class MainApp extends StatelessWidget {
  MainApp({super.key});

  final QuillController controller = QuillController(
      document: Document(),
      selection: const TextSelection(baseOffset: 0, extentOffset: 0));

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      theme: ThemeData.light(useMaterial3: false),
      home: Scaffold(
        body: Center(
          child: QuillSimpleToolbar(
              configurations:
                  QuillSimpleToolbarConfigurations(controller: controller)),
        ),
      ),
    );
  }
}

```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

https://github.com/singerdmx/flutter-quill/assets/66178504/e2a88221-6e95-40c6-8f2a-3fe0c20a6dd8

</details>

### Logs

<details><summary>Logs</summary>

```console
Restarted application in 124ms.
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
The following assertion was thrown during layout:
A RenderFlex overflowed by 28 pixels on the right.

The relevant error-causing widget was:
  Row
  Row:file:///home/rodrigo/.pub-cache/hosted/pub.dev/flutter_quill-9.0.1/lib/src/widgets/toolbar/buttons/font_family_button.dart:279:14

The overflowing RenderFlex has an orientation of Axis.horizontal.
The edge of the RenderFlex that is overflowing has been marked in the rendering with a yellow and
black striped pattern. This is usually caused by the contents being too big for the RenderFlex.
Consider applying a flex factor (e.g. using an Expanded widget) to force the children of the
RenderFlex to fit within the available space instead of being sized to their natural size.
This is considered an error condition because it indicates that there is content that cannot be
seen. If the content is legitimately bigger than the available space, consider clipping it with a
ClipRect widget before putting it in the flex, or using a scrollable container rather than a Flex,
like a ListView.
The specific RenderFlex in question is: RenderFlex#2b9ca relayoutBoundary=up2 OVERFLOWING:
  creator: Row ← Padding ← IconTheme ← Builder ← Align ← SizedBox ← Padding ← ConstrainedBox ←
    Listener ← RawGestureDetector ← GestureDetector ← Semantics ← ⋯
  parentData: offset=Offset(10.0, 0.0) (can use size)
  constraints: BoxConstraints(0.0<=w<=14.0, 0.0<=h<=11.2)
  size: Size(14.0, 11.2)
  direction: horizontal
  mainAxisAlignment: spaceBetween
  mainAxisSize: min
  crossAxisAlignment: center
  textDirection: ltr
  verticalDirection: down
◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤◢◤
════════════════════════════════════════════════════════════════════════════════════════════════════
Another exception was thrown: A RenderFlex overflowed by 54 pixels on the right.
Another exception was thrown: A RenderFlex overflowed by 46 pixels on the right.

```

</details>

**Comment by freshtechtips** [comment]
Thank you for the report. I can confirm this bug is happening

**Comment by freshtechtips** [comment]
Fixed, please take a look at `9.0.2`

Thank you once again, if you have any questions feel free to ask.
